### DISCUSSION TITLE
How to skip build failure and continue to scan project

### DISCUSSION THREAD

**Discussion by ccc1994** [JetBrains/Qodana#265] [Q&A] [answered]
I am scanning many projects that were not developed by me, how to skip failure like maven import failure and continue to scan proejct.
Error message:
`o.j.q.s.i.r.QodanaInspectionApplication - Maven project ${projectName} has import problems`

**Comment by brichbash** [comment, marked as answer]
Hello,

It is required to load your project to analyze your JVM projects. So in this case, skipping the import is not an option. If the projects use dependencies from private repositories, you can pass your `settings.xml` to `/root/.m2/settings.xml` in the Qodana container, e.g.: `qodana scan -v path/to/your/settings.xml:/root/.m2/settings.xml`
